src/services/utils/date-helpers.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+
+/**
+ * Takes a date (either a moment or Date object) and returns an ISO date string in UTC time
+ * If there is no date, return an empty string
+ * Throws a TypeError if an invalid value is passed in
+ *
+ * @param {String|Object} date
+ *
+ * @returns {String}
+ */
+function _formatDate (date) {
```

Comment:
If this is meant to be private, it should be marked that way in the comment. Otherwise, let’s get rid of that `_`.